cmd/cmd.js

```diff
@@ -264,12 +264,21 @@ class Cmd {
   test() {
     program
       .command('test [file]')
-      .option('-n , --node [node]', __('Node to connect to (default: vm)'))
+      .option('-n , --node <node>', __('Node to connect to. Valid values are ["vm", "embark", "<custom node endpoint>"]: \n') + 
+              '                       vm - ' + __('Starts an Ethereum simulator (ganache) and runs the tests using the simulator') + '\n' +
+              '                       embark - ' + __('Uses the node associated with an already running embark process') + '\n' + 
+              '                       ' + __('<custom node endpoint> - Connects to a running node available at the end point and uses it to run the tests'), 
+              /^(vm|embark|((ws|https?):\/\/([a-zA-Z0-9_.-]*):?([0-9]*)?))$/i, null)
```

Comment:
It can be http (withotu the "s").

Reply:
In the regex? The `s?` should handle that. Means `s` should appear 0 or 1 times

Reply:
Oh my bad. I blame sleepy me.